src/components/PaymentStripe/checkoutForm.js

```diff
@@ -0,0 +1,370 @@
+import React, { useState } from 'react';
```

Comment:
@TahimiLeonBravo @LeandroTorresSicilia should we name the component `StripeCheckout` instead?